### PR TITLE
[1LP][RFR] More containers tests

### PR DESCRIPTION
**Pull request by gshefer**
{{pytest: cfme/tests/containers/test_start_page.py  cfme/tests/containers/test_reports.py cfme/tests/containers/test_configurable_menus.py -v --use-provider cm-env1 }}

- Added cfme/tests/containers/test_start_page.py - test_start_page
- Added cfme/tests/containers/test_reports.py - test_container_reports_base_on_options
- Few page match fixes

### CODE REVIEW

File: cfme/tests/containers/test_configurable_menus.py [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+    if (yaml_config['product']['datawarehouse_manager'] == is_visible) and \
+            (yaml_config['prototype']['monitoring'] == is_visible):
+        return  # If this is already the state, return
+
+    yaml_config['product']['datawarehouse_manager'] = is_visible
+    yaml_config['prototype']['monitoring'] = is_visible
+
+    appliance.set_yaml_config(yaml_config)
+    appliance.reboot()
+    ensure_browser_open()
```
Comment (mfalesni) [inline cfme/tests/containers/test_configurable_menus.py]: You probably want to do the browser quit here. @RonnyPfannschmidt what is the approach with the browser manager here?

Comment (gshefer) [inline cfme/tests/containers/test_configurable_menus.py]: Why browser quit?, I actually want to ensure it open after the reboot finished.

Comment (RonnyPfannschmidt) [inline cfme/tests/containers/test_configurable_menus.py]: as far as i can tell browser quit or ensure open is not actually needed

the reboot functiopn of appliance by default waits for the web ui to reappear - so i believe just a call to reboot is sufficient in this case

Comment (gshefer) [inline cfme/tests/containers/test_configurable_menus.py]: Well, I think it's working without the ensure_browser_open.... I'm removing it and let's see the results...

**Comment by gshefer** [comment]
@RonnyPfannschmidt @mfalesni 
ensure_browser_open has removed.
PRT results are True positive.

**Comment by mshriver** [comment]
@gshefer please squash commits here and check all remaining commits are signed. Thanks!

File: cfme/containers/provider/__init__.py [hunk 2]
```diff
@@ -256,6 +256,9 @@ def num_image_registry_ui(self):
 class All(CFMENavigateStep):
     prerequisite = NavigateToAttribute('appliance.server', 'LoggedIn')
 
+    def am_i_here(self):
+        return match_page()
```
Comment (mshriver) [inline cfme/containers/provider/__init__.py]: **Required**
Add a `summary='Pods'` attribute so we get a 3-way correlation for match_location.  controller and title attributes match what's in the header script, summary matches the displayed page title.

File: cfme/tests/containers/test_configurable_menus.py [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+    yaml_config['prototype']['monitoring'] = is_visible
+
+    appliance.set_yaml_config(yaml_config)
+    appliance.reboot()
+
+
+def is_menu_visible(appliance, link_text):
+    navigate_to(ContainersProvider, 'All')
+    logged_in_page = appliance.browser.create_view(BaseLoggedInPage)
+    return link_text in logged_in_page.navigation.nav_links()
```
Comment (mshriver) [inline cfme/tests/containers/test_configurable_menus.py]: **Optional**
Consider a stronger assertion against `logged_in_page.navigation.currently_selected`, a property which returns a list of selected menu items.  Here you would assert that currently_selected returns `['Compute', 'Containers', 'Providers']`

**Pull request merged**